### PR TITLE
docs: Added fleet vs kubernetes comparison table

### PR DESCRIPTION
**Pull request by kayrus**
removed unnecessary/odd fleetd options from TLS example
/cc @jonboulle resolves https://github.com/coreos/fleet/issues/1405

### CODE REVIEW

**Comment by kayrus** [comment]
TODO: Add some extra description and move comparison table into a separate markdown page.

**Comment by kayrus** [comment]
@jonboulle updated PR.
@robszumski @joshix can you take a look at this PR?

**Comment by kayrus** [comment]
I'd like to add some extra info on how does kubelet run containers and how fleet does. From my point of view k8s uses API to communicate with docker and systemd units to spawn containers with rkt. Is it true? I can not find doc which explains this question. Should I put a link to [this](https://github.com/kubernetes/kubernetes/blob/master/docs/getting-started-guides/rkt/notes.md) doc?

**Comment by jonboulle** [comment]
/cc @yifan-gu 
I'm not sure this is documented outside of the code, though it should be.

File: Documentation/fleet_vs_kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# fleet vs kubernetes Comparison Table
+
+Here is a short comparison table which will help you to choose which tool is more suitable for you:
```
Comment (joshix) [inline Documentation/fleet_vs_kubernetes.md]: This table briefly compares `fleet` to Kubernetes to help inform deployment decisions:

**Comment by yifan-gu** [comment]
> From my point of view k8s uses API to communicate with docker and systemd units to spawn containers with rkt. Is it true?

For rkt+k8s integration. k8s relies on systemd to start/monitor/stop rkt containers. (k8s creates a systemd service file that has `ExecStart=rkt run`). Does this answer you question ? @kayrus

**Comment by kayrus** [comment]
@yifan-gu was my assumption about docker API correct? @jonboulle told me that kubelet uses `docker run`.

**Comment by jonboulle** [comment]
No, I was wrong, it uses the api now.

kayrus <notifications@github.com> schrieb am Mi., 2. März 2016 22:07:

**Comment by kayrus** [comment]
Added info on how k8s runs containers

**Comment by jonboulle** [comment]
@joshix got time for a review?

**Comment by joshix** [comment]
Yep. Tonight or tomorrow. I actually have a rewrite branch in progress, but
saw that @kayrus was going to add these new k8s details, so I should only
be a merge and a commit or two away.

**Comment by joshix** [comment]
@kayrus @jonboulle Reviewed, rewritten, superseded in #1478

**Pull request closed without merging**